script2.sh

```diff
@@ -35,6 +35,7 @@ elif [ $tool = "zstd" ]
 	do
 		for((level = 1; level <= 19; level++))
 		do
+		#cache being cleared after each level of compression
```

Comment:
before merging ur PR, I merged one more.
so if u want to raise a PR, pull and then change and then raise PR.